packages/api/src/controllers/access-control.ts

```diff
@@ -119,4 +128,19 @@ app.post(
   }
 );
 
+async function fireEventWebhook(
+  req: Request,
+  content: DBStream | WithID<Asset>
+) {
+  const queue = req.queue;
+  await queue.publishWebhook(`events.playback.accessControl`, {
```

Comment:
I noticed that in the scoping doc we mention adding a context object - is this something we plan on adding in a future iteration?
<img width="730" alt="Screenshot 2023-03-14 at 9 27 05 AM" src="https://user-images.githubusercontent.com/32714304/225072550-66e63346-4aa5-403f-81ac-3f4cbfc261ae.png">

Reply:
This is a different thing, the screen from the notion doc is referring to the webhook that Studio is going to call during the `gate` check (https://github.com/livepeer/studio/pull/1665), this pr is referring instead to user configured webhooks events that notify access control triggers